keyboards/macro3/rules.mk

```diff
@@ -17,3 +17,4 @@ BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
 ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

Comment:
Cannot be enabled at the keyboard level, as it currently causes configurator style builds to fail.

Reply:
Noted and updated accordingly to keymap level.